Allow lablgtk2 to run without requiring all extra libraries to be installed.

--- origsrc/lablgtk-2.14.0/src/Makefile	2010-02-14 13:37:41.101228500 -0600
+++ src/lablgtk-2.14.0/src/Makefile	2010-02-14 18:13:20.669237700 -0600
@@ -267,7 +267,7 @@ lablgtk2: Makefile $(CONFIG) lablgtk2.in
 lablgtk2: Makefile $(CONFIG) lablgtk2.in
 	sed -e "s|@INSTALLDIR@|$(INSTALLDIR)|g" \
 	    -e "s|@LABLGLDIR@|$(LABLGLDIR)|g" \
-	    -e "s|@MLLIBS@|$(MLLINK) $(MLLIBS)|g" \
+	    -e "s|@MLLIBS@|$(MLLIBS)|g" \
 	    -e "s|@INITOBJS@|$(INITOBJS)|g" \
 	    -e "s|@THOBJS@|$(THOBJS)|g" \
 	    -e "s|@THINITOBJS@|$(THINITOBJS)|g" \
--- origsrc/lablgtk-2.14.0/src/lablgtk2.in	2005-11-27 23:13:52.000000000 -0600
+++ src/lablgtk-2.14.0/src/lablgtk2.in	2010-02-14 18:14:24.913912300 -0600
@@ -16,6 +16,7 @@ cmd="$0"
 
 if test -n "@LABLGLDIR@"; then
   libpath="-I @LABLGLDIR@"
+  libraries="lablgl.cma"
 fi
 
 while test -n "$1"; do
@@ -48,7 +49,11 @@ fi
 libpath="-I $lablgtkdir $libpath"
 
 if test $extra = yes; then
-    libraries="@MLLIBS@"
+    for extralib in @MLLIBS@ ; do
+        if test -f $lablgtkdir/$extralib ; then
+            libraries="$libraries $extralib"
+        fi
+    done
 else
     libraries="lablgtk.cma"
 fi
